src/main/jbake/content/blog/2016/05/24/hawkular-btm-ticketmonster-demo.adoc

```diff
@@ -0,0 +1,23 @@
+= Obtaining business metrics without the need to instrument your application
+Gary Brown
+2016-05-24
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, btm, apm, business metrics, distributed tracing
+
+
+There are many solutions available to allow a business to instrument its applications to report relevant business metrics. However most (if not all) of these solutions require the application to be modified to extract the information and report it to the chosen management solution.
+
+What happens if you need the flexibility to begin recording and analysing new business metrics immediately? Even with the current trend towards continuous delivery, there is still a delay while the business analyst creates a feature request, and the development team understands the requirement, makes the modifications and pushes the changes through the pipeline.
+
+Would it not be better for business analysts to be able to react immediately to emerging requirements, which may also be temporary (i.e. short lived), to collect and analyse the required business metrics?
+
+This https://vimeo.com/167739840/cd36d4ff8d[_Hawkular BTM_ demo] shows how to capture business metrics from an executing business application without having to introduce changes to the application or any frameworks being used. The business metrics can be presented in the form of charts, and also used within filters to help a user to focus in on the important information required to analyse how a business is functioning. The business metrics are organised based on the concept of a _"Business Transaction"_.
```

Comment:
Perhaps inline the video instead of "This .. demo ..". Makes the page more interesting to look at